src/core/server/server.ts

```diff
@@ -65,7 +65,7 @@ import type { RequestHandlerContext, PrebootRequestHandlerContext } from '.';
 
 const coreId = Symbol('core');
 const rootConfigPath = '';
-const KIBANA_STARTED_EVENT = 'kibana_started';
+const KIBANA_STARTED_EVENT = 'kibana-started';
```

Comment:
This is a breaking change for an existing event. Any reason why we prefer `-` over `_`. AFAIK, the ES text analyzer does not break words by `_` but it does for `-`. That's the only reason I preferred underscores.

However, I'd rely on @elastic/platform-analytics for any recommendations around naming conventions.

Reply:
cc breaking changes - we're in a very early stage with this data and I'm not too scared to make a concious breaking change now. Long term concistency IMO is an important win.

However, the text analyzer is interesting argument!
I will change all event of them use an underscore then.

Reply:
As far as I could see the `event_type` field is mapped as a `keyword` and this means it is not analyzed. So word breaks should not matter or am I missing something?